Tests/Behaviors/CoverFlowLayoutBehaviorTests.swift

```diff
@@ -68,12 +68,12 @@ class CoverFlowLayoutBehaviorTests: XCTestCase {
         let cell2 = brickView.cellForItem(at: IndexPath(item: 1, section: 1))
         let cell3 = brickView.cellForItem(at: IndexPath(item: 2, section: 1))
 
-        XCTAssertEqualWithAccuracy(cell1!.transform.scaleX, 0.5, accuracy: 0.01)
-        XCTAssertEqualWithAccuracy(cell1!.transform.scaleY, 0.5, accuracy: 0.01)
-        XCTAssertEqualWithAccuracy(cell2!.transform.scaleX, 5/6, accuracy: 0.01)
-        XCTAssertEqualWithAccuracy(cell2!.transform.scaleY, 5/6, accuracy: 0.01)
-        XCTAssertEqualWithAccuracy(cell3!.transform.scaleX, 5/6, accuracy: 0.01)
-        XCTAssertEqualWithAccuracy(cell3!.transform.scaleY, 5/6, accuracy: 0.01)
+        XCTAssertEqual(cell1!.transform.scaleX, 0.5, accuracy: 0.01)
+        XCTAssertEqual(cell1!.transform.scaleY, 0.5, accuracy: 0.01)
+        XCTAssertEqual(cell2!.transform.scaleX, 5/6, accuracy: 0.01)
+        XCTAssertEqual(cell2!.transform.scaleY, 5/6, accuracy: 0.01)
+        XCTAssertEqual(cell3!.transform.scaleX, 5/6, accuracy: 0.01)
+        XCTAssertEqual(cell3!.transform.scaleY, 5/6, accuracy: 0.01)
```

Comment:
I think you might be able to use `?` instead of `!` for these, so that if the cell is nil it wont crash the test runner. can you try changing that and seeing if it works?

Reply:
Nope, that gives a error, All the prams have to be not optional on `XCTAssertEqual` when using accuracy

Reply:
My preference would be to do `let cell1 = brickView.cellForItem(at: IndexPath(item: 0, section: 1))!` instead of adding `cell!` over and over. The problem, however is that it would crash if there's no cell instead of triggering a test failure (which is arguably a bit more graceful). So even better would be to `guard let cell1 = .. else { XCTFail() }` because you can provide a message.